README.md

```diff
@@ -28,30 +28,20 @@ and Astro, designed to fit into your workflow.
 
 ## Status
 
-We're working on it! Things are experimental at the moment. And, we don't have
-docs yet.
+We're working on it! Things are experimental at the moment.
 
 But if you're curious and brave, check out
-[keystatic.thinkmill.com.au](https://keystatic.thinkmill.com.au) to try it out
+[keystatic.com](https://keystatic.com) to try it out
 (and check out our growing collection of templates)
```

Comment:
```suggestion
(and check out our growing collection of templates).
```

Just adding a lil' full stop. Rest looks good but you will need to run prettier over this file as it is failing the prettier check.

Reply:
Yep, updated the README directly on GitHub — now I have pulled this branch locally to run prettier 👍